microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/security/ListenerSecurityConfigurationTest.java

```diff
@@ -0,0 +1,239 @@
+        Operations operations = new Operations(onlineManagementClient);
+        Address undertowAddress = Address.subsystem("undertow").and("server", "default-server").and("https-listener",
+                "https");
+        Batch batch = new Batch();
+        batch.undefineAttribute(undertowAddress, "ssl-context");
+        batch.writeAttribute(undertowAddress, "security-realm", "ApplicationRealm");
+        ModelNodeResult result = operations.batch(batch);
+        result.assertSuccess();
+
+        result = onlineManagementClient.execute("/subsystem=elytron/server-ssl-context=httpsSSC:remove()");
+        result.assertSuccess();
+
+        result = onlineManagementClient.execute(
+                "/subsystem=elytron/key-manager=httpsKM:remove()");
+        result.assertSuccess();
+
+        result = onlineManagementClient.execute(
+                "/subsystem=elytron/key-store=httpsGenKS:remove()");
+        result.assertSuccess();
+
+        new Administration(onlineManagementClient).reload();
+
+        Files.delete(keyStoreFile);
+    }
+
+    private static void disableHTTPListener() throws InterruptedException, TimeoutException, IOException, CliException {
+        ModelNodeResult result = onlineManagementClient.execute(
+                "/subsystem=undertow/server=default-server/http-listener=default:remove");
+        result.assertSuccess();
+
+        result = onlineManagementClient.execute(
+                "/subsystem=remoting/http-connector=http-remoting-connector:write-attribute(name=connector-ref, value=https)");
+        result.assertSuccess();
+
+        new Administration(onlineManagementClient).reload();
+    }
+
+    private static void enableHTTPListener() throws InterruptedException, TimeoutException, IOException, CliException {
+        ModelNodeResult result = onlineManagementClient.execute(
+                "/subsystem=undertow/server=default-server/http-listener=default:add(" +
+                        "socket-binding=http,redirect-socket=https,enable-http2=true)");
+        result.assertSuccess();
+
+        result = onlineManagementClient.execute(
+                "/subsystem=remoting/http-connector=http-remoting-connector:write-attribute(name=connector-ref, value=default)");
+        result.assertSuccess();
+
+        new Administration(onlineManagementClient).reload();
+    }
+
+    /**
+     * @tpTestDetails Test to verify that the {@code openapi} endpoint is available via HTTPS once the listener
+     *                has been properly configured
+     * @tpPassCrit The {@code openapi} endpoint can be reached via [https] schema at related standard port number
+     *             (8443), response status is HTTP 200 and response body is not empty
+     *
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    public void testOpenApiOnHTTPSListener() {
+        io.restassured.RestAssured.useRelaxedHTTPSValidation();
```

Comment:
is this relaxed https validation reset anywhere ?

if not then this test can cause side effect == all https checks will be relaxed
Atm we can live with it, but it must be at least tracked in issues

Reply:
No, definitely not resetting it. You're right, fixing...